user/cron-jobs.md

```diff
@@ -4,6 +4,8 @@ layout: en
 permalink: /user/cron-jobs/
 ---
 
+> Update 2016-08-02: We will not enable cron jobs for new organizations/repositories for the time being and we unfortunately don't have an ETA regarding when we will restart enabling them. Sorry for the inconvenience.
```

Comment:
Needs to stand out more. With quoting, the text might be too light to draw attention.